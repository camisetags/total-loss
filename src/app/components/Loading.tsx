import * as React from 'react';

const Loading = () => <div className="loading"> Carregando... </div>;

export default Loading;
