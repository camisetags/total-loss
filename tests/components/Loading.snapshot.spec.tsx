import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Loading from '../../src/app/components/Loading';

describe('Loading', () => {
  it('renders Loading component', () => {
    const rendered = renderer.create(<Loading />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
