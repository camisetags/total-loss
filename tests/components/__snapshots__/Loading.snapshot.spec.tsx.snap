// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders Loading component 1`] = `
<div
  className="loading"
>
   Carregando... 
</div>
`;
